clar2wasm/tests/wasm-generation/comparison.rs

```diff
@@ -0,0 +1,33 @@
+use clar2wasm::tools::crosscheck_compare_only;
+use clarity::vm::Value;
+use proptest::proptest;
+
+use crate::string_ascii;
+
+fn filter_invalid_char(input: &Value) -> Value {
+    let invalid_chars: [char; 1] = ['\''];
+    let v: String = input.clone().expect_ascii().replace(invalid_chars, "");
+
+    Value::Sequence(clarity::vm::types::SequenceData::String(
+        clarity::vm::types::CharType::ASCII(clarity::vm::types::ASCIIData { data: v.into() }),
+    ))
+}
```

Comment:
Why is this needed? Can’t we use `'` in clarity?

Reply:
The issue is that `\'` is an invalid char.